WNPRC_EHR/src/org/labkey/wnprc_ehr/pages/husbandry/WaterCalendar.jsp

```diff
@@ -1155,7 +1150,7 @@
                             window.open(newWaterOrder,'_blank');
 
                         }
-                        if (!response.success){
+                        else if (!response.success){
```

Comment:
this could just be an else

Reply:
it  was giving me some problems when it was just an if not sure why